consensus/core/src/core.rs

```diff
@@ -299,7 +299,7 @@ pub(crate) struct CoreSignals {
 
 impl CoreSignals {
     #[allow(dead_code)]
-    pub(crate) fn new() -> (Self, CoreSignalsReceivers) {
+    pub fn new() -> (Self, CoreSignalsReceivers) {
```

Comment:
why is this no longer gated via pub(crate)?

Reply:
well It's just that there is no need as the `CoreSignals` is already declared as `pub(crate) CoreSignals`. The visibility level of `new` is already limited by its parent